spdx-utils/src/test/kotlin/SpdxDocumentModelTest.kt

```diff
@@ -85,4 +88,19 @@ class SpdxDocumentModelTest : WordSpec({
             serializedJson shouldBe formatJson(json)
         }
     }
+
+    "SPDX document" should {
+        "fail when no relationship of type DESCRIBES or documentDescribes is contained" {
+            val yaml = readResourceAsText(
+                "/spdx-spec-examples/SPDXYAMLExample-2.2-no-relationship-describes-or-document-describes.spdx.yml"
+            )
+
+            val exception = shouldThrow<ValueInstantiationException> {
+                SpdxModelMapper.fromYaml(yaml, SpdxDocument::class.java)
+            }
+
+            exception.message shouldContain "Document must have at least one relationship of type DESCRIBES " +
```

Comment:
Why not `shouldBe`?

Reply:
Because the message also contains: "Cannot construct instance of `org.ossreviewtoolkit.spdx.model.SpdxDocument`, problem:" and I didn't want to put that in `shouldBe` since it is the problem message I am interested in.